app/models/tenant.rb

```diff
@@ -18,6 +18,8 @@ class Tenant < ApplicationRecord
   default_value_for :divisible,   true
   default_value_for :use_config_for_attributes, false
 
+  before_destroy :ensure_can_be_destroyed
```

Comment:
was this moved here for a specific reason?

Reply:
ok, comment looks confusing, I was trying to show that ```has_ancestry```  defines ```before_destroy: :apply_orphan_strategy``` for the same object